pkg/pillar/containerd/containerd.go

```diff
@@ -635,6 +645,9 @@ func (client *Client) Resolver(ctx context.Context) (resolver.ResolverCloser, er
 // CtrNewUserServicesCtx returns a new user service containerd context
 // and a done func to cancel the context after use.
 func (client *Client) CtrNewUserServicesCtx() (context.Context, context.CancelFunc) {
+	if base.IsHVTypeKube() {
```

Comment:
Can we set the variable `ctrdServicesNamespace` in the `init()` clause, and then this remains unchanged?

Reply:
Sure, I'll make the change

Reply:
Would it be acceptable to call the existing GetServicesNamespace() in CtrNewUserServicesCtx and CtrNewUserServicesCtxWithLease?